ksrates/fc_wgd.py

```diff
@@ -802,6 +802,35 @@ def _run_iadhore(config_file):
 
     return
 
+def compute_weights_anchor_pairs(df, min_ks=0.05, max_ks=20, aln_id=0, aln_len=300,
+        aln_cov=0):
+    """
+    Modified from wgd.
+    Computes the weights of anchor pair Ks estimates.
+    
+    :param min_ks: minimum Ks value considered (hard coded to 0.05 Ks)
```

Comment:
```suggestion
    :param min_ks: minimum Ks value considered (default 0.05 Ks)
```

Reply:
The reason why this is currently hard-coded is not here. It is a parameter here, i.e., here it can be changed and is thus not hard-coded here. The reason it is hard-coded is that elsewhere, where the function is called, it is not parameterized anymore and thus the default value given here is used.